neuvector-monitor-5.3.yaml

```diff
@@ -1,11 +1,13 @@
 package:
   name: neuvector-monitor-5.3
   version: 5.3.2
-  epoch: 0
+  epoch: 1
   description: NeuVector Full Lifecycle Container Security Platform.
   copyright:
     - license: Apache-2.0
   dependencies:
+    runtime:
+      - consul-1.16-oci-entrypoint
```

Comment:
that'd be great if we could explain why do we need this with couple of comments I guess? What do you think?